release.sh

```diff
@@ -391,18 +391,18 @@ if [ "$BUILD_TYPE" = "Train" ]; then
   IS_BIG_RELEASE=`expr "$PRIVATE_DIFF_SIZE" \> 400`
   if [ "$IS_BIG_RELEASE" = "1" ]; then
     echo "The diff for the private release is pretty big. You may want to add the following comment to that PR:"
-    echo
-    echo "> There's no need to review every line of this diff, since it includes every commit from the release. Instead you can do the following:"
-    echo ">"
-    echo '> ```'
-    echo "> git fetch origin $TRAIN_BRANCH"
-    echo "> git fetch $PRIVATE_REMOTE $PRIVATE_BRANCH"
-    echo "> git checkout -b $PRIVATE_BRANCH $PRIVATE_REMOTE/$PRIVATE_BRANCH"
-    echo "> git diff origin/$TRAIN_BRANCH"
-    echo '> ```'
-    echo ">"
-    echo "> That diff should show only the changes from the private repo, proving that the $TRAIN_BRANCH branches are equal in other respects."
-    echo
+    echo "COPY HERE ------------------------------------------->"
```

Comment:
Fwiw, my original thinking was that this could be copy/pasted as is, including the markdown for quoting it. Then it's naturally delimited from the surrounding text without needing the extra delimiter lines.